docs/source/contents/4_draft_mapping.rst

```diff
@@ -2152,6 +2152,622 @@ Drop all values in mods:part.
+
+Justification
+^^^^^^^^^^^^^
+
+XPath
+^^^^^
+:code:`relatedItem[not(@*)]` OR
+:code:`relatedItem[@type='host'][@displayLabel[matches(., 'project') or matches(., 'Project') or matches(., 'Digital Collection') or matches(., 'Project Part') or matches(., 'Is Part Of')]` OR
+:code:`relatedItem/abstract`
+
+Decision
+^^^^^^^^
+Do not migrate.
+
+relatedItem[@type='host'][@displayLabel='Collection']
+-----------------------------------------------------
+Use Case
+^^^^^^^^
+This XPath is used to indicate the resource's archival collection.
+
+Justification
+^^^^^^^^^^^^^
+
+XPath
+^^^^^
+:code:`relatedItem[@type='host'][@displayLabel='Collection']`
+
+Decision
+^^^^^^^^
+The `dbo:collection` property was selected.
+
+`Example record - heilman:261 <https://digital.lib.utk.edu/collections/islandora/object/heilman:261/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <relatedItem type="host" displayLabel="Project">
+      <titleInfo>
+        <title>Botanical Photography of Alan S. Heilman</title>
+      </titleInfo>
+    </relatedItem>
+    <relatedItem type="host" displayLabel="Collection">
+      <titleInfo>
+        <title>Botany Department Photographs</title>
+      </titleInfo>
+      <identifier type="local">AR.0488</identifier>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dbo: <http://dbpedia.org/ontology/> .
+
+    <https://example.org/objects/1> dbo:collection "Botany Department Photographs, AR.0488" .
+
+relatedItem[@type='series'][@displayLabel='Project']
+----------------------------------------------------
+Use Case
+^^^^^^^^
+The `@type='series'` XPath indicates a resource's archival series.
+
+Justification
+^^^^^^^^^^^^^
+XPath
+^^^^^
+:code:`relatedItem[@type='series'][@displayLabel='Project']`
+
+Decision
+^^^^^^^^
+The `opaque:memberOfArchivalSeries` predicate was chosen for these values.
+
+`Example record - roth:1538 <https://digital.lib.utk.edu/collections/islandora/object/roth:1538/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <relatedItem type="series" displayLabel="Project">
+      <titleInfo>
+        <title>Series II: Margaret Ann Roth Photographs and Other Materials, 1947 March 11-2002 December 14 (bulk 1947 March 11-1955 March 20). Sub-Series A: Photographs, 1947 March 11-1955 March 139</title>
+      </titleInfo>
+    </relatedItem>
+    <relatedItem displayLabel="Collection" type="host">
+      <titleInfo>
+        <title>A. G. "Dutch" and Margaret Ann  Roth  Papers</title>
+      </titleInfo>
+      <identifier>MS.3334</identifier>
+    </relatedItem>
+    <relatedItem displayLabel="Project" type="host">
+      <titleInfo>
+        <title>Albert "Dutch" Roth Photograph Collection</title>
+      </titleInfo>
+    </relatedItem>
+
+.. code-block:: turtle
+
+    @prefix dbo: <http://dbpedia.org/ontology/> .
+    @prefix opaque: <http://opaquenamespace.org/ns/> .
+
+    <https://example.org/objects/1> dbo:collection """A. G. "Dutch" and Margaret Ann Roth Papers, MS.3334""" ;
```

Comment:
Thinking on some of the decisions we made in Mat's PR with shelfLocator, I'm not sure that we really want to retain this level of description about the archival collection. I don't think it hurts to have it, but it is possible that the Archival Series may be rearranged in the future and that this information will become inaccurate. I feel that this information doesn't really need to be duplicated in our metadata - it can live in ArchivesSpace by itself. Let me know if others have thoughts.

Reply:
As mentioned earlier, I agree with removing this section (and not migrating) - it makes our approach to handling archival metadata more consistent.